tox.ini

```diff
@@ -18,7 +18,7 @@ setenv =
     py36,py37,py38,py39: PY_MAJOR_VERSION = py3
 commands =
     py27,py36,py38,py39: pytest --cov=petl petl
-    py37: nosetests -v --with-coverage --cover-package=petl --with-doctest --doctest-options=+NORMALIZE_WHITESPACE petl -I"csv_py2\.py" -I"db\.py"
+    py37: pytest --cov=petl petl
```

Comment:
I think we don't need to split `py37` anymore because you solved/removed the previous hack here. Right?

Reply:
You are right https://github.com/petl-developers/petl/pull/584/commits/9ff34359bc5d38f971943f526bc226a06f8d8cea